### PR TITLE
Fixes: #3278: Add java docs to methods which have it missing

### LINKED ISSUE #3278: Add java docs to methods which have it missing
Recently we merged a few PRs with java docs missing and it would be great to have java docs added for all new methods that were added in those PRs. 

Or in general, it would be great to have java docs added to methods that have it missing.

### PR DESCRIPTION
**Pull request by kbhardwaj123**
**Added Javadocs wherever missing**

Fixes #3278  Add java docs to methods which have it missing

**What changes did you make and why?**

Added javadocs to the files which did not have them

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/actions/PageEditClient.java [hunk 1]
```diff
@@ -5,6 +5,9 @@
 
 import io.reactivex.Observable;
 
+/**
+ * Uses the PageEditInterface to make edit calls to MW API
+ */
 public class PageEditClient {
```
Comment (maskaravivek) [inline app/src/main/java/fr/free/nrw/commons/actions/PageEditClient.java]: Most of the java docs for methods in this class are not descriptive. Either make it descriptive or remove it if you are not sure what it does. Please check the same for other classes as well. :)

Comment (kbhardwaj123) [reply app/src/main/java/fr/free/nrw/commons/actions/PageEditClient.java]: Sure I'll make them more descriptive

Comment (maskaravivek) [reply app/src/main/java/fr/free/nrw/commons/actions/PageEditClient.java]: Great. Ping here once you are done.

Comment (kbhardwaj123) [reply app/src/main/java/fr/free/nrw/commons/actions/PageEditClient.java]: @maskaravivek Sir done :)

**Comment by maskaravivek** [comment]
Thanks for your contribution @kbhardwaj123 :)

**Pull request merged**